interop-future/jvm/src/main/scala/scalaz/zio/interop/Future.scala

```diff
@@ -0,0 +1,209 @@
+package scalaz.zio
+
+import scalaz.zio.internal.impls.Env
+import scala.concurrent.ExecutionContext
+import scala.util.{ Failure, Success, Try }
+import scala.reflect.ClassTag
+
+package object future {
+
+  /**
+   * A [[scala.concurrent.Future]] is a running computation, and corresponds
+   * most closely to ZIO `Fiber`.
+   */
+  type Future[+A] = Fiber[Throwable, A]
+
+  private val Global = ExecutionContext.Implicits.global
+
+  private final def unsafeRun[E, A](ec: ExecutionContext, io: IO[E, A]): A =
+    Env.fromExecutionContext(ec).unsafeRun(io)
+
+  private final def toTry[A](e: Either[Throwable, A]): Try[A] =
+    e.fold(Failure(_), Success(_))
+
+  /**
+   * An API-compatible implementation of [[scala.concurrent.Future]], which
+   * is backed by ZIO. While this structure is not performant, due to emulation
```

Comment:
I think that it would be very helpful to add this Future shim to the benchmarks (whenever Future is also used).

That way, end-users can make an informed decision on wether to use the shim or just go for the full rewrite.

Reply:
If performance of Future matters, do the full rewrite! The shim is only intended for minimal use of Future where the dominating factor is IO or CPU, not thread pool overhead.